index.html

```diff
@@ -0,0 +1,166 @@
+        <div class="main-program-title">
+            <p>Main Program</p>
+            <hr class="hr-line">
+        </div>
+        <div class="programs-card-container mb-3 pt-3">
+            <div class="program">
+                <img src="img/program-icon-1.png" alt="#" class="program-img">
+                <p class="program-title">Lecture</p>
+                <p class="program-description">Listen to speaker from various countries about the messages of sharing
+                    and opening.</p>
+            </div>
+            <div class="program">
+                <img src="img/program-icon-2.png" alt="#" class="program-img">
+                <p class="program-title">CS Exhibition</p>
+                <p class="program-description">Appreciate various creations Applying CC license of artists, organized
+                    from art center Nabi.</p>
+            </div>
+            <div class="program">
+                <img src="img/program-icon-3.png" alt="#" class="program-img">
+                <p class="program-title">Forum</p>
+                <p class="program-description">Have time to share your thoughts and opinions with experts for each
+                    topic.</p>
+            </div>
+            <div class="program">
+                <img src="img/program-icon-4.png" alt="#" class="program-img">
+                <p class="program-title">Workshop</p>
+                <p class="program-description">Try creating your own work using open source license rather than just
+                    watching at it.</p>
+            </div>
+            <div class="program">
+                <img src="img/program-icon-5.png" alt="#" class="program-img">
+                <p class="program-title">CS Ignite</p>
+                <p class="program-description">Get opportunities to network with CC Affiliates around the worls, also
+                    after the summit.</p>
+            </div>
+        </div>
+        <div class="see-the-whole-program mt-5 mb-5">
+            <p>
+                <a class="" href="#">SEE THE WHOLE PROGRAM</a>
+            </p>
+        </div>
+
+        <button class="join-cs mb-3">Join CS African Summit 2023</button>
+    </section>
+
+       <!-- Features section -->
+    <section id="featured-speakers" class="featured-speakers mb-1 py-4">
+
+    </section>
+
+         <!-- partners section -->
+    <section class="partners mb-3 py-5 none">
+        <div class="partners-header">
+            <h3 class="partner">Partner</h3>
+            <hr class="hr-line">
+        </div>
+
+        <div class="partner-imgs py-4">
+            <img src="img/google.png" alt="" class="partner-img">
+            <img src="img/mozilla.png" alt="" class="partner-img">
+            <img src="img/ibm.png" alt="" class="partner-img">
+            <img src="img/flutter.png" alt="" class="partner-img">
+            <img src="img/airbnb.png" alt="" class="partner-img">
+        </div>
+    </section>
```

Comment:
as suggested previously 
Well done 👏 adding the partner's section, but it should stick to the Figma design, which means that properties like colour, fonts, paddings, margins, etc should be the same. Kindly fix that 😄

your design
![image](https://github.com/Edgahkipkemoi/capstone/assets/60612329/9373d99c-df3e-4fb2-b491-fc8496329369)

cindy's design
![image](https://github.com/Edgahkipkemoi/capstone/assets/60612329/04f77bb8-4bec-4866-989f-d4a203cf1b7d)